public/controllers/management/components/management/ruleset/actions-buttons.js

```diff
@@ -93,15 +93,15 @@ class WzRulesetActionButtons extends Component {
    * @param {Array} files
    * @param {String} path
    */
-  async uploadFiles(files, resource) {
+  async uploadFiles(files, resource, overwrite) {
     try {
       let errors = false;
       let results = [];
       const rulesetHandler = new RulesetHandler(resource);
       for (let idx in files) {
         const { file, content } = files[idx];
         try {
-          await rulesetHandler.updateFile(file, content, resource !== RulesetResources.LISTS); // True does not overwrite the file
+          await rulesetHandler.updateFile(file, content, overwrite); // True does not overwrite the file
```

Comment:
Does that comment make sense?

Reply:
It seems that the comment has no sense. We could remove it.

Reply:
Solved